bentoml/models.py

```diff
@@ -44,9 +44,17 @@ def delete(
 
 @inject
 def import_model(
-    path: str, *, _model_store: "ModelStore" = Provide[BentoMLContainer.model_store]
+        path: str,
+        input_format: t.Optional[str] = None,
+        *,
+        protocol: t.Optional[str] = None,
+        user: t.Optional[str] = None,
+        passwd: t.Optional[str] = None,
+        params: t.Optional[t.Dict[str, str]] = None,
+        subpath: t.Optional[str] = None,
+    _model_store: "ModelStore" = Provide[BentoMLContainer.model_store]
 ) -> Model:
-    return Model.from_fs(fs.open_fs(path)).save(_model_store)
+    return Model.import_from(path, input_format, protocol=protocol, user=user, passwd=passwd, params=params, subpath=subpath).save(_model_store)
```

Comment:
same here - also in this case it's especially confusing - user may think they can import any model files, e.g. a Tensorflow SavedModel or a sklearn pickle file, we should explain that this only works for models exported from BentoML

Reply:
I'll rename it to `import_bento_model`, maybe?

Reply:
I feel adding some docstring here should be good enough, I'm afraid `import_bento_model` may lead to users thinking it must be models packed in a Bento.  

One thing missing here is we don't seem to check if the path being imported is a valid exported model, maybe it should check that a `bentomodel.yaml` file exist in the path before importing? That way we can show a good error message telling user that import target must be a model exported from BentoML model store.